scripts/clib/clib_test.py

```diff
@@ -0,0 +1,81 @@
+import os
+import argparse
+import shutil
+import subprocess
+
+def main():
+    parser = argparse.ArgumentParser(description="Test cpython library")
+    parser.add_argument("-cp", nargs=1, default="../../../cpython", required=False, help="Local cpython path.")
+    parser.add_argument("-rp", nargs=1, default="../..", required=False, help="Local RustPython path.")
+    args = vars(parser.parse_args())
+    CPYTHONPATH = args["cp"]
+    RPYTHONPATH = args["rp"]
+
+    if isinstance(CPYTHONPATH, list):
+        CPYTHONPATH = CPYTHONPATH[0]
+    if isinstance(RPYTHONPATH, list):
+        RPYTHONPATH = RPYTHONPATH[0]
+
+    liblst = list()
+
+    with open("clib_list.txt", 'r') as f:
+        for line in f.readlines():
+            line = line[:line.find('#')]
+            line = line.strip()
+            if line:
+                liblst.append(line)
+
+    open("clib_out.txt", 'w').close()
+
+    for lib in liblst:
+        lib_exist = False
+        test_exist = False
+        test_do = True
+        message = [f"{lib}:"]
+
+        if os.path.isfile(f"{RPYTHONPATH}/Lib/{lib}.py"):
+            lib_exist = True
+            os.rename(f"{RPYTHONPATH}/Lib/{lib}.py", f"{RPYTHONPATH}/Lib/{lib}_tmp_mv.py")
```

Comment:
Rather than renaming original library files, redefining `RUSTPYTHONPATH` will be fine. 

This is the logic I think.

1. cp -r Lib LibTest  (but in python, not shell)
1. cp ../cpython/Lib/x LibTest/x
1. RUSTPYTHONPATH=LibTest rustpython Lib/test/test_...

Reply:
Sure, that would be easy. But I am not fully grasping why that would be a better design. Is it because messing with library files via script may open room for possible human mistakes?

Reply:
I've tried to use RUSTPYTHONPATH environment variable to control which library files are referenced by rustpython interpreter but I seem to be doing something wrong, as it's working as if it's still referencing files in Lib, not LibTest. Could you guide me through how I could do what you suggested in python?

Reply:
I tried to set environment variable through os.environ. Let me try again with a new method and see how it turns out.